### PR TITLE
docs: updated documentation and setup for mainline development.

### PR DESCRIPTION
**Pull request by CBrunet2212**
GitHub Issue: #

## Proposed Changes

Documentation content changes

## What is the current behavior?

The PR template is old.
The documentation doesn't contain a page for breaking changes.
There is no badge in the README for library version and download count.
Dependabot is used.
Version numbering is not configured using MainLine.

## What is the new behavior?

The PR template is updated.
The documentation contains a page for breaking changes.
There are in the README for library version and download count.
Dependabot is removed.
Version numbering is configured to use MainLine.

## Checklist

Please check if your PR fulfills the following requirements:

- [X] Documentation has been added/updated
- [ ] ~~Automated Unit / Integration tests for the changes have been added/updated~~
- [X] Contains **NO** breaking changes
- [ ] ~~Updated the Changelog~~
- [ ] ~~Associated with an issue~~

## Other information

Changes based on those made in the MessageDialogService [commit with the same goal ](https://github.com/nventive/MessageDialogService/commit/f3b1aa145f2b371a2d8bebaec42d049074d1d5de?diff=split)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,9 +128,6 @@ public sealed class MainActivity : Microsoft.UI.Xaml.ApplicationActivity
 Note that when you run your app in debug from Visual Studio (or other IDEs), the new SplashScreen icon doesn't show.
 It shows when you run the app from the launcher (even debug builds).
```
Comment (CBrunet2212) [inline README.md:130]: Wondering if there are any additional features we want to detail here that not covered in the Getting Started section?

Comment (Soap-141) [reply README.md:130]: To be honest I would remake it like this one https://github.com/nventive/ReviewService

Comment (Soap-141) [reply README.md:130]: @jeanplevesque

Comment (jeanplevesque) [reply README.md:130]: It's fine like this. :)

Also, I don't think there are other features to document.

**Review by jeanplevesque — APPROVED** [review]

**Review by Soap-141 — APPROVED** [review]

**Pull request merged**